pilot/cmd/pilot-agent/main.go

```diff
@@ -346,25 +346,25 @@ var (
 			// Legacy - so pilot-agent can be used with citadel node agent.
 			// Main will be replaced by istio-agent when we clean up - this code can stay here and be removed with the rest.
 			sdsUDSPath := sdsUdsPathVar.Get()
-			sdsEnabled, sdsTokenPath := detectSds(controlPlaneBootstrap, sdsUDSPath, trustworthyJWTPath)
+			nodeAgentSDSEnabled, sdsTokenPath := detectSds(controlPlaneBootstrap, sdsUDSPath, trustworthyJWTPath)
```

Comment:
nit: why is `nodeAgent...` a better name? Doesn't SDS imply node agent?

Reply:
No. Istiod is removing the need for node agent - SDS is already in pilot-agent.